packages/css/src/components/form/form-group.scss

```
@@ -0,0 +1,220 @@
+@use "sass:math";
+
+@use "form.variables" as formvariables;
+
+.form-group {
+  + .form-group:not(.dso-static),
+  + .dso-form-buttons {
+    margin-top: formvariables.$dso-form-group-margin;
+  }
+
+  &:not(.dso-static) + .form-group.dso-static,
+  &.dso-static + .form-group:not(.dso-static) {
+    margin-top: formvariables.$dso-form-group-margin * 0.5;
+  }
+
+  .form-group.dso-edit + & {
```

We willen liever geen suffixing ampersands. Vaak doen we er iets slims mee (denken we), maar in dit geval vermoed ik dat het gewoon luiheid is.

```suggestion
  .form-group.dso-edit + .form-group {
```

Natuurlijk wel op root niveau van de stylesheet

---

`&.dso-edit + .form-group {`